tests/sles4sap/hana_install.pm

```diff
@@ -255,6 +269,10 @@ sub run {
     }
     assert_script_run "df -h";
 
+    # Run restorecon again on SLES for SAP 16.0 and newer as we have created and mounted new
+    # FS since the last run
+    restorecon_rootfs() if is_sle('16+');
```

Comment:
How about using the `has_selinux` function in version_utils ? in this way test are independent from the product version.

Reply:
Wasn't aware of that. Will use it.